editor/index.tsx

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter, Route } from 'react-router-dom';
 import Editor from './editor';
 import './index.css';
+import 'higlass/dist/hglib.css';
```

Comment:
Can just import the CSS here which `css-loader` takes care of..

Reply:
Sounds good!